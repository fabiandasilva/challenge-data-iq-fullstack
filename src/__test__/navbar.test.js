/**
 * @jest-environment jsdom
 */
import { render } from "@testing-library/react";
import React from "react";
import Navbar from "../components/NavBar";

test("Navbar se renderiza correctamente", () => {
  const { asFragment } = render(<Navbar />);
  expect(asFragment()).toMatchSnapshot();
});
