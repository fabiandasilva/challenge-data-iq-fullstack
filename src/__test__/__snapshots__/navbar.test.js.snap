// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar se renderiza correctamente 1`] = `
<DocumentFragment>
  <nav
    class="px-1.5 py-4 sm:p-6 w-full bg-white drop-shadow-sm"
  >
    <div
      class="max-w-7xl mx-auto flex justify-between items-center"
    >
      <a
        href="/"
      >
        <img
          alt="Data Iq"
          data-nimg="1"
          decoding="async"
          height="56"
          loading="lazy"
          src="/_next/image?url=%2Fimg.jpg&w=256&q=75"
          srcset="/_next/image?url=%2Fimg.jpg&w=128&q=75 1x, /_next/image?url=%2Fimg.jpg&w=256&q=75 2x"
          style="color: transparent;"
          width="110"
        />
      </a>
      <div
        class="w-34 sm:w-80 flex items-center h-10 rounded-md border bg-white/20 px-1 sm:px-3 py-2 text-xs"
      >
        <span
          class="text-gray-500  dark:text-gray-400 border-r-2 mr-1"
        >
          <svg
            class="mr-0 sm:mr-2 h-4 w-5 items-center stroke-slate-500 sm:h-5 sm:w-5"
            fill="none"
            viewBox="0 0 24 24"
          >
            <path
              d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z"
            />
          </svg>
        </span>
        <span
          class="hidden sm:block w-full "
        >
          <input
            class="flex-grow bg-transparent outline-none w-full text-sm"
            placeholder="Buscar por nombre, email o ciudad..."
            type="text"
            value=""
          />
        </span>
        <span
          class="sm:hidden"
        >
          <input
            class="flex-grow bg-transparent outline-none"
            placeholder="Buscar..."
            type="text"
            value=""
          />
        </span>
      </div>
    </div>
  </nav>
</DocumentFragment>
`;
